src/components/PetitionSidebar/index.js

```diff
@@ -11,14 +11,15 @@ import settings from 'settings';
 
 const PetitionSidebar = ({
   processing,
+  closed,
   timeMetric,
   isSupportable,
   userHasSupported,
   startDate,
   runningTime
 }) => (
   <aside role='complementary' className={styles.root}>
-    {processing &&
+    {processing || closed &&
```

Comment:
no, because we have two different messages depending on if the letter response is processing, or if the the editors reviewed the response (closed) :(